hrms/payroll/doctype/payroll_entry/payroll_entry.json

```diff
@@ -312,12 +274,56 @@
    "options": "Draft\nSubmitted\nCancelled\nQueued\nFailed",
    "print_hide": 1,
    "read_only": 1
+  },
+  {
+   "fieldname": "column_break_5",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "column_break_21",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "section_break_24",
+   "fieldtype": "Section Break"
+  },
+  {
+   "fieldname": "section_break_26",
+   "fieldtype": "Section Break"
+  },
+  {
+   "fieldname": "column_break_35",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "column_break_13",
+   "fieldtype": "Column Break"
+  },
+  {
+   "depends_on": "eval:doc.docstatus==0",
+   "fieldname": "get_employees",
+   "fieldtype": "Button",
+   "label": "Get Employees"
+  },
```

Comment:
I understand that this button is associated with the table but IMO, moving this button inside the form has made it more confusing. Primary action was more intuitive. Maybe we should keep the employee table on page 1 itself, and revert back to the primary action.

Reply:
Added `scroll_to_field` instead


https://user-images.githubusercontent.com/24353136/210774601-8b0dce6a-1155-4d54-b358-b4f25b9851c8.mp4